### PR TITLE
DiskRP sdk changes for api version 2020-05-01

### PR DESCRIPTION
**Pull request by chetmaddula**

### CODE REVIEW

**Comment by hyonholee** [comment]
build failed. Please make it sure build succeeded locally and all playback tests pass

**Comment by isra-fel** [comment]
@hyonholee Hi Hyonho I've been seeing compute PRs going into different branches recently. For example #12509 is targeting `compute-2020-06` while this one is for `master`.
What's your team's plan for PR reviewing? I assume every compute PR will be reviewed just by you and go to your branch, and eventually a big PR will be created from compute to master. Is that so?

File: eng/mgmt/mgmtmetadata/compute_resource-manager.txt [hunk 1]
```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=D:\hylee-sdk\may\sdk
-2020-05-26 00:22:02 UTC
+cmd.exe /c autorest.cmd https://github.com/chetmaddula/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=D:\GitSwagger\azure-sdk-for-net\sdk
+2020-06-05 03:46:10 UTC
 Azure-rest-api-specs repository information
-GitHub fork: Azure
+GitHub fork: chetmaddula
```
Comment (hyonholee) [inline eng/mgmt/mgmtmetadata/compute_resource-manager.txt:9]: regenerate after your Swagger PR is merged to Azure repo

**Comment by hyonholee** [comment]
@isra-fel this PR is for DiskRP.   This is separate from Compute-2020-06, which contains CRP API updates.

**Pull request closed without merging**